locale/en-US/navbarheight.properties

```diff
@@ -1,10 +1,10 @@
-nav_bar_height=Navigation Bar height
```

Comment:
This file belongs to en-US localization (as it can bee seen from the path 'locale/en-US/navbarheight.properties')